addons/l10n_in/views/res_company_view.xml

```diff
@@ -8,7 +8,10 @@
         <field name="inherit_id" ref="base.view_company_form"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='vat']" position="attributes">
-                <attribute name="string" attrs="{'invisible': [('country_code', '!=', 'IN')]}">GSTIN</attribute>
+                <attribute name="attrs">{'invisible': [('country_code', '=', 'IN')]}</attribute>
+            </xpath>
+            <xpath expr="//field[@name='vat']" position="after">
+                <field name="vat" string="GSTIN" attrs="{'invisible': [('country_code', '!=', 'IN')]}"/>
```

Comment:
Well currently if l10n_in is installed, vat field will be renamed to GSTIN for every other company, so I thought it would be nice to fix it even though it wasn't in the task. But I will revert it once I come back

Reply:
It won't be working if other l10n want to do the same.

Reply:
I don't think you should go back on your initial intention to fix the `GSTIN` issue :slightly_smiling_face:
We should either remove the `GSTIN` override, or implement it correctly using @william-andre 's advice :wink: 
Or is it outside the scope of this task (doesn't feel like it) ?

Reply:
Okay, I think I misunderstood and thought this had to be reversed. 
Probably it is better to remove the override?
(The task only described the bank journal field btw, I just noticed this by accident)

Reply:
I would remove it :shrug: 
To be fixed in master, see the field `vat_label` in https://github.com/odoo/odoo/pull/79659

Reply:
removed it, will fix in master